pkg/storage/types/types.go

```diff
@@ -57,6 +57,7 @@ type ImageStore interface { //nolint:interfacebloat
 	PutIndexContent(repo string, index ispec.Index) error
 	StatIndex(repo string) (bool, int64, time.Time, error)
 	GetBlobContent(repo string, digest godigest.Digest) ([]byte, error)
+	GetBlobReader(repo string, digest godigest.Digest) (io.ReadCloser, int64, error)
```

Comment:
It is not clear how this is different from the method at line 51

```suggestion
        VerifyBlobDigestValue(repo string, digest godigest.Digest) error
```

Let's call this function from the scrub logic.